aliroot.sh

```diff
@@ -11,6 +11,8 @@ build_requires:
   - DAQ:slc6.*
 env:
   ALICE_ROOT: "$ALIROOT_ROOT"
+prepend_path:
+  ROOT_INCLUDE_PATH: "$ALIROOT_ROOT/include"
```

Comment:
So, first off, this of course has not to be merged yet. This is required for the QA on SWAN, see [ATO-383](https://alice.its.cern.ch/jira/browse/ATO-383) (and the addition of this export can be limited to ROOT 6-only builds because of that).